### PR TITLE
Collectd: multiple changes as requested in #217

### PR DESCRIPTION
**Pull request by mimugmail**
- Add more precise description to FQDNLookup
- Second validator for network collector (IP and hostname)
- Add progress to Save button

Please review @fabianfrz @fichtner

### CODE REVIEW

**Comment by mimugmail** [comment]
Yes, if you have a multitenant setup where hostname is always just fw, you can set it to customer-fw. So this woud be the name on the central server.

File: net-mgmt/collectd/src/opnsense/mvc/app/controllers/OPNsense/Collectd/forms/general.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <id>general.fqdnlookup</id>
         <label>FQDN lookup</label>
         <type>checkbox</type>
-        <help>This will enable the FQDNLookup option.</help>
+        <help>If Hostname is determined automatically this setting controls whether or not the daemon should try to figure out the "fully qualified domain name", FQDN. This is done using a lookup of the name returned by gethostname.</help>
```
Comment (fabianfrz) [inline net-mgmt/collectd/src/opnsense/mvc/app/controllers/OPNsense/Collectd/forms/general.xml]: probably write FQDN into round brackets as it is an abbreviation and remove the quotes.

"This is done by looking up the name by using gethostname."

Comment (fabianfrz) [inline net-mgmt/collectd/src/opnsense/mvc/app/controllers/OPNsense/Collectd/forms/general.xml]: maybe hostname lowercase and maybe add "the"

**Comment by mimugmail** [comment]
@fabianfrz Just copied from the website, did the suggested changes. Thanks

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**